docs/source/optimizations/ibm/ibmq_dd_advanced.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>We could probably shorten this a bit. I'd suggest cutting out everything before "There are many possible ways to determine..." and adding a sentence at the beginning to introduce the notebook's topics. The sentence could be something simple like:</p><p> </p><p>"In this notebook, we will examine different ways Superstaq places dynamical decoupling (DD) sequences within circuit idle windows.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/770/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Need to update this but I think this PR is good to go after @knsmith